packages/main/src/CustomListItem.ts

```diff
@@ -1,43 +1,13 @@
 import { isTabNext, isTabPrevious } from "@ui5/webcomponents-base/dist/Keys.js";
+import { ClassMap } from "@ui5/webcomponents-base/dist/types.js";
```

Comment:
Maybe `import type { ClassMap }`
as you have done for the  I18nText:

import type { I18nText } from "@ui5/webcomponents-base/dist/i18nBundle.js";